modules/sno-about-sno-cluster-worker-nodes.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: CONCEPT
+[id="sno-about-sno-cluster-worker-nodes_{context}"]
+= About worker nodes on single-node OpenShift clusters
+
+Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. However, because there is only a single control plane node, ingress traffic is routed differently compared to standard cluster configurations with three control plane nodes.
```

Comment:
```suggestion
Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. However, because there is only a single control plane node, ingress traffic is, by default, routed differently compared to standard cluster configurations with three control plane nodes.
```